demo/src/app/components/+pagination/demos/disable-direction-links/disable-direction-links.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-pagination-disable-direction-links',
+  templateUrl: './disable-direction-links.html'
```

Comment:
since this demo shows disabled and enabled direction links, it's better to be named  just `direction links`